identity_service/main.py

```diff
@@ -16,7 +17,8 @@ async def serve():
         IdentityServiceImpl(), server
     )
     server.add_insecure_port("0.0.0.0:8080")
-    await Db().connect()
+    await PostgresClient().connect()
+    RedisClient()
```

Comment:
It's doubtful, but okay.

Reply:
Here i just renamed the class. Seems fine to me